docs/development/installing_from_source.rst

```diff
@@ -38,9 +38,9 @@ General Notes
 
      apk add --virtual build-deps file make gcc musl-dev libffi-dev
 
-  On Fedora Rawhide for Fedora 33, one might need to do this::
+  On Fedora Rawhide or Fedora 33, one might need to do this::
```

Comment:
```suggestion
  On Fedora Rawhide for Fedora 33, one might need to do this::
```

I believe the "for" was correct there. This is documenting one specific system version, not all versions of Rawhide.